src/components/Textarea/index.js

```diff
@@ -175,26 +186,35 @@ class TextArea extends PureComponent {
         return (
             <Wrapper className={this.props.className}>
                 {this.props.topLabel && <TopLabel>{this.props.topLabel}</TopLabel>}
-                <StyledTextarea
-                    spellCheck="false"
-                    autoCorrect="off"
-                    autoCapitalize="off"
-                    maxRows={this.props.maxRows}
-                    maxLength={this.props.maxLength}
-                    rows={this.props.rows}
-                    className={this.props.className}
-                    disabled={this.props.isDisabled}
-                    name={this.props.name}
-                    style={this.props.customStyle}
-                    onFocus={this.props.onFocus}
-                    onBlur={this.props.onBlur}
-                    value={this.props.value}
-                    readOnly={this.props.readOnly}
-                    onClick={this.props.autoSelect ? event => event.target.select() : null}
-                    placeholder={this.props.placeholder}
-                    onChange={this.props.onChange}
-                    border={this.getColor(this.props.state)}
-                />
+                <TextAreaWrapper>
+                    {this.props.state && (
+                        <StyledIcon
```

Comment:
The state icon was removed for a reason. Why did you add it back?https://github.com/trezor/trezor-ui-components/pull/86

Reply:
fixed